### PR TITLE
DS-3631_validation_form

### PR DESCRIPTION
**Pull request by MatiasRoy-Rodriguez**
1. Create form validation
2. Create form validation syles
3. Add new country service
4. Fix time script

### CODE REVIEW

File: html/js/script.js [hunk 1]
```diff
@@ -1,46 +1,148 @@
...
+		let numberRegex = new RegExp("^[0-9]*$");
+		if (!phone.match(numberRegex)) {
+			return true;
+		}
+		return false;
+	}
+
+
+	function _validateForm(form) {
+console.log(form)
```
Comment (hcardoso81) [inline html/js/script.js]: eliminar el console

File: html/js/script.js [hunk 2]
```diff
@@ -1,46 +1,148 @@
+"use strict";
 document.addEventListener('DOMContentLoaded', () => {
-	const earlyForm = document.getElementById('earlyForm');
-	earlyForm.addEventListener('submit', sendData);
+	const earlyForms = document.querySelectorAll('form');
+	const dialCodeContainer = document.querySelector('.iti__selected-dial-code');
+	earlyForms.forEach(form => { form.addEventListener('submit', sendData) });
 
 	function sendData(e) {
+		console.log('se llama')
```
Comment (hcardoso81) [inline html/js/script.js]: eliminar el console

File: html/js/modules/intlTelInput/phoneValidation.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	$(phoneInputFieldFooter).closest('.holder').removeClass('error');
+};
+
+// on blur: validate2
+phoneInputFieldFooter.addEventListener('blur', function () {
+	console.log('se llama')
+	reset();
+	if (phoneInputFieldFooter.value.trim()) {
+		if (phoneInputFooter.isValidNumber()) {
+			console.log('no tendria que entrar aca')
```
Comment (hcardoso81) [inline html/js/modules/intlTelInput/phoneValidation.js]: eliminar console

**Review by ornella-688 — APPROVED** [review]

**Comment by ornella-688** [comment]
Good!

**Review by camila-stefano-ms — APPROVED** [review]
Good!

**Review by hcardoso81 — APPROVED** [review]
Good!

File: html/js/script.js (hunk 1 above)
Comment (MatiasRoy-Rodriguez) [reply html/js/script.js]: Ya estaba eliminado [chor(js): remove console log](https://github.com/FromDoppler/emms22/pull/24/commits/a1e7ef63d5646f81b0a236556b55fe1176062fcb)

File: html/js/script.js (hunk 2 above)
Comment (MatiasRoy-Rodriguez) [reply html/js/script.js]: Ya estaba eliminado [chor(js): remove console log](https://github.com/FromDoppler/emms22/pull/24/commits/a1e7ef63d5646f81b0a236556b55fe1176062fcb)

File: html/js/modules/intlTelInput/phoneValidation.js (hunk 3 above)
Comment (MatiasRoy-Rodriguez) [reply html/js/modules/intlTelInput/phoneValidation.js]: Ya estaba eliminado [chor(js): remove console log](https://github.com/FromDoppler/emms22/pull/24/commits/a1e7ef63d5646f81b0a236556b55fe1176062fcb)

**Pull request merged**